<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>drrTool</class>
 <widget class="QDialog" name="drrTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
	Based on a template by Emanuel Claesson (c) 2013 with modifications by Paris Flood

	Licensed under the Apache License, Version 2.0 (the &quot;License&quot;);
	you may not use this file except in compliance with the License.
	You may obtain a copy of the License at

		http://www.apache.org/licenses/LICENSE-2.0

	Unless required by applicable law or agreed to in writing, software
	distributed under the License is distributed on an &quot;AS IS&quot; BASIS,
	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
	See the License for the specific language governing permissions and
	limitations under the License.
*/

/*
	COLOR_DARK     = #191919
	COLOR_MEDIUM   = #353535
	COLOR_MEDLIGHT = #5A5A5A
	COLOR_LIGHT    = #DDDDDD
	COLOR_BLUE  = #2158AA
	COLOR_ORANGE = #D66C23
*/
* {
	background: #191919;
	color: #DDDDDD;
border: 1px solid #5A5A5A;
}

 QMenuBar {
	background: #191919;
	border-style: none;
 }

 QMenuBar::item {
	 background: transparent;
     spacing: 3px; /* spacing between menu bar items */
     padding: 5px 25px;
 }

 QMenuBar::item:selected{
	background: #353535;
  	border: 2px solid #D66C23;
    border-style: none none solid none;
}

 QMenuBar::item:pressed {
	background: #5A5A5A;
}

QCheckBox, QRadioButton {
	border: none;
}

QRadioButton::indicator, QCheckBox::indicator {
	width: 13px;
	height: 13px;
}

QRadioButton::indicator::unchecked, QCheckBox::indicator::unchecked {
	left: 5px;
	border: 1px solid #5A5A5A;
	background: none;
}

QRadioButton::indicator:unchecked:hover, QCheckBox::indicator:unchecked:hover {
	left: 5px;
	border: 1px solid #D66C23;
}

QRadioButton::indicator::checked, QCheckBox::indicator::checked {
	left: 5px;
	border: 1px solid #5A5A5A;
	background: #2158AA;
}

QRadioButton::indicator:checked:hover, QCheckBox::indicator:checked:hover {
	left: 5px;
	border: 1px solid #D66C23;
	background: #2158AA;
}

QGroupBox::title {
	subcontrol-origin: margin;
    subcontrol-position: top center;
}

QScrollBar {
	border: 1px solid #5A5A5A;
	background: #191919;
}

QScrollBar:horizontal {
	height: 15px;
	margin: 0px 16px 0px 16px;
}

QScrollBar:vertical {
	width: 15px;
	margin: 16px 0px 16px 0px;
}

QScrollBar::handle {
	background: #353535;
	border: 1px solid #5A5A5A;
}

QScrollBar::handle:horizontal {
	border-width: 0px 1px 0px 1px;
}

QScrollBar::handle:vertical {
	border-width: 1px 0px 1px 0px;
}

QScrollBar::handle:horizontal {
	min-width: 20px;
}

QScrollBar::handle:vertical {
	min-height: 20px;
}

QScrollBar::add-line, QScrollBar::sub-line {
	background:#353535;
	border: 1px solid #5A5A5A;
	subcontrol-origin: margin;
}

QScrollBar::add-line {
	position: absolute;
}

QScrollBar::add-line:horizontal {
	width: 15px;
	subcontrol-position: right;
	left: 15px;
}

QScrollBar::add-line:vertical {
	height: 15px;
	subcontrol-position: bottom;
	top: 15px;
}

QScrollBar::sub-line:horizontal {
	width: 15px;
	subcontrol-position: left;
}

QScrollBar::sub-line:vertical {
	height: 15px;
	subcontrol-position: top;
}

QScrollBar:left-arrow, QScrollBar::right-arrow, QScrollBar::up-arrow, QScrollBar::down-arrow {
	border: 1px solid #5A5A5A;
	width: 3px;
	height: 3px;
}

QScrollBar::add-page, QScrollBar::sub-page {
	background: none;
}

QAbstractButton {
	background: #252525;
}

QAbstractButton:disabled {
	background: #3F3F3F;
}

QAbstractButton:hover {
	background: #353535;
}

QPushButton:hover {
	background: #353535;
  	border: 1px solid #D66C23;
}

QAbstractButton:pressed {
	background: #5A5A5A;
}

QAbstractItemView {
	show-decoration-selected: 1;
 	outline: 0;
	selection-background-color: #2158AA;
	selection-color: #DDDDDD;
	alternate-background-color: #353535;
}

QHeaderView {
	border: 1px solid #5A5A5A;
}

QHeaderView::section {
	background: #191919;
	border: 1px solid #5A5A5A;
	padding: 4px;
}

QHeaderView::section:selected, QHeaderView::section::checked {
	background: #353535;
}

QTableView {
	gridline-color: #5A5A5A;
}

QTabBar {
	margin-left: 2px;
}

QTabBar::tab {
	border-radius: 0px;
	padding: 4px;
	margin: 4px;
}

QTabBar::tab:selected {
	background: #353535;
}

QComboBox::down-arrow {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QComboBox::drop-down {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QComboBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #5A5A5A;
}

QAbstractSpinBox {
	padding-right: 15px;
}

QAbstractSpinBox::up-button, QAbstractSpinBox::down-button {
	border: 1px solid #5A5A5A;
	background: #353535;
	subcontrol-origin: border;
}
QAbstractSpinBox::up-button:hover, QAbstractSpinBox::down-button:hover {
	border: 1px solid #D66C23;
}
QAbstractSpinBox::up-button:pressed, QAbstractSpinBox::down-button:pressed {
	background: #5A5A5A;
}

QAbstractSpinBox::up-arrow, QAbstractSpinBox::down-arrow {
	width: 3px;
	height: 3px;
	border: 1px solid #5A5A5A;
}
QAbstractSpinBox::up-arrow:hover, QAbstractSpinBox::down-arrow:hover {
	border: 1px solid #D66C23;
}

QSlider {
	border: none;
}

QSlider::groove:horizontal {
	height: 5px;
	margin: 4px 0px 4px 0px;
}

QSlider::groove:vertical {
	width: 5px;
	margin: 0px 4px 0px 4px;
}

QSlider::handle {
	border: 1px solid #5A5A5A;
	background: #353535;
}

QSlider::handle:horizontal {
	width: 15px;
	margin: -4px 0px -4px 0px;
}

QSlider::handle:vertical {
	height: 15px;
	margin: 0px -4px 0px -4px;
}

QSlider::add-page:vertical, QSlider::sub-page:horizontal {
	background: #2158AA;
}

QSlider::sub-page:vertical, QSlider::add-page:horizontal {
	background: #353535;
}

QLabel {
	border: none;
}

QProgressBar {
	text-align: center;
}

QProgressBar::chunk {
	width: 1px;
	background-color: #2158AA;
}

QMenu::separator {
	background: #353535;
}

QGroupBox {
	font: bold
}

QListWidget::item:selected {
    background-color: #2158AA;
}

 QMenu::item:selected{
	background: #353535;
	border: 2px solid #D66C23;
    border-style: none solid none none;
}

 QMenu::pressed {
	background: #5A5A5A;
}

QMenu::separator {
    height: 5px;
	background: #2158AA;
}</string>
  </property>
  <widget class="QLabel" name="drr_image_label">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>650</width>
     <height>650</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="maxThreshGroupBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>700</y>
     <width>931</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>MINIMUM THRESHOLD</string>
   </property>
   <widget class="QDoubleSpinBox" name="minUpperSpinBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-1000000000000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="minLowerSpinBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-1000000000000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="minLowerPrefix">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Bound:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minUpperPrefix">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Bound:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValuePrefix">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Threshold Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="minValue">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QSlider" name="minSlider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>461</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="maxThreshGroupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>830</y>
     <width>931</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>MAXIMUM THRESHOLD</string>
   </property>
   <widget class="QDoubleSpinBox" name="maxUpperSpinBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-1000000000000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000000.000000000000000</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="maxLowerSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="minimum">
     <double>-1000000000000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="maxLowerPrefix">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lower Bound:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxUpperPrefix">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Upper Bound:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxValuePrefix">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Threshold Value:</string>
    </property>
   </widget>
   <widget class="QLabel" name="maxValue">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VALUE</string>
    </property>
   </widget>
   <widget class="QSlider" name="maxSlider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>461</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>681</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
